ui/src/components/icons/PencilSettingsIcon.tsx

```diff
@@ -0,0 +1,19 @@
+import { IconProps } from './icon';
+
+export default function PencilSettingsIcon({ className }: IconProps) {
+  return (
+    <svg
+      className={className}
+      xmlns="http://www.w3.org/2000/svg"
+      viewBox="0 0 18 18"
```

Comment:
seems like a weird size you may want to re-export making sure you get the surrounding frame should either be 16 or 24